File: projects/Examples/java_example/files/reading.java
class Read
{
  public static void number()
  {

    Scanner scan = new Scanner(System.in); 

    System.out.println("Reading first number");
    Integer number1 = scan.nextInt();  

    System.out.println("Reading second number");
    Integer number2 = scan.nextInt(); 

    Integer soma = number1 + number2;


    System.out.println("Total: " + soma);  
  }
}

File: projects/Examples/c_example/files/reading.c
int reading(){

  int num1, num2, soma;
  
  printf("Reading first number...");
  scanf("%i", &num1);
  
  printf("Reading second number...");
  scanf("%i", &num2);
  
  soma = num1 + num2;
  
  printf( "Total: %i\n", soma );

  return 0;
}
